controllers/operatorpolicy_status.go

```diff
@@ -775,6 +794,11 @@ func buildDeploymentCond(
 		message = fmt.Sprintf("the deployments %s do not have their minimum availability", names)
 	}
 
+	if !canModifyComp {
+		status = metav1.ConditionTrue
+		message += " - ignore, compliance overridden by complianceConfig"
```

Comment:
Can we add a conditional to only add this message if it is overriding the status? It looks to me like it would be adding this even if all of the deployments are available.